api/operator.go

```diff
@@ -323,3 +325,14 @@ func (op *Operator) Metrics(q *QueryOptions) ([]byte, error) {
 
 	return metricsBytes, nil
 }
+
+// MetricsSummary returns a MetricsSummary struct and query metadata
+func (op *Operator) MetricsSummary(q *QueryOptions) (*metrics.MetricsSummary, *QueryMeta, error) {
```

Comment:
`github.com/armon/go-metrics` brings in a bunch of other dependencies, since the only parts of the metrics package we need are the structs to to return here, I think it'd be better to just copy the structs and in the api/ package so we can remove the go-metrics dependency

Reply:
Separated away from operator.go as suggested previously, and included only the required structs.